stubs/protobuf/google/protobuf/descriptor_pb2.pyi

```diff
@@ -220,119 +220,121 @@ global___ExtensionRangeOptions = ExtensionRangeOptions
 class FieldDescriptorProto(google.protobuf.message.Message):
     """Describes a field within a message."""
     DESCRIPTOR: google.protobuf.descriptor.Descriptor = ...
-    class Type(_Type, metaclass=_TypeEnumTypeWrapper):
-        pass
     class _Type:
-        V = typing.NewType('V', builtins.int)
-    class _TypeEnumTypeWrapper(google.protobuf.internal.enum_type_wrapper._EnumTypeWrapper[_Type.V], builtins.type):
+        ValueType = typing.NewType('ValueType', builtins.int)
+        V: typing_extensions.TypeAlias = ValueType
+    class _TypeEnumTypeWrapper(google.protobuf.internal.enum_type_wrapper._EnumTypeWrapper[_Type.ValueType], builtins.type):
         DESCRIPTOR: google.protobuf.descriptor.EnumDescriptor = ...
-        TYPE_DOUBLE = FieldDescriptorProto.Type.V(1)
+        TYPE_DOUBLE: FieldDescriptorProto.Type.ValueType = ...  # 1
```

Comment:
Nit: If possible, please drop the `= ...`.

Reply:
It's called `FieldDescriptorProto._Type` not `Type` as far as I understand?

Reply:
Can drop the `= ...` in the next version of mypy-protobuf (3.2.0). (These files are autogenerated)

Type inherits from `_Type`. It should work. It actually does work if you put the file anywhere else. Haven't figured out why it's behaving differently.

Reply:
The unit tests for `mypy-protobuf` all pass - and they have similar constructs. Mypy is somehow behaving differently when operating inside the stubs/protobuf directory - and I'm having trouble figuring out why.

Reply:
Did you try reordering the file so that `Type` is defined before _TypeEnumTypeWrapper? You could try manually for now, just to see whether that helps.

Reply:
Dropped the `= ...` in https://github.com/dropbox/mypy-protobuf/pull/326